Userland/Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -548,6 +548,25 @@ BigInt* Value::to_bigint(GlobalObject& global_object) const
     }
 }
 
+// 7.1.15 ToBigInt64 ( argument ), https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tobigint64
+i64 Value::to_bigint_int64(GlobalObject& global_object) const
+{
+    auto* bigint = to_bigint(global_object);
+    if (global_object.vm().exception())
+        return INVALID;
+    // FIXME: This is probably wrong.
+    return static_cast<i64>(bigint->big_integer().to_u64());
```

Comment:
this is not wrong. `to_u64` of SignedBigInteger returns a 2nd complement representation of the number for the negative case, making this cast always correct.

Reply:
Cool, fixed :+1: